### ISSUE TITLE
False Positive: You are not currently using the "strict-transport-security" header...

### ISSUE THREAD

**Issue by jat-247** [HTTPSChecker/releases#30]
The warning is showing up as an Issue for a site I'm scanning:
"You are not currently using the "strict-transport-security" header, which can leave you open to man-in-the-middle attacks with tools like SSLStrip"

The .htaccess file has the following set and active with no caching (server-side or client side):
Header add Strict-Transport-Security "max-age=31415926;includeSubDomains;"

I believe this is a false positive; followup diagnostics/logs to confirm?

**Comment by stilliard** [comment]
Hey @jat-247 
Thanks for report, sounds like a false positive but would you be able to verify it with curl or similar tool on the terminal, or using a site such as https://onlinecurl.com/
If possible, could you provide the url of the site?
Thank you

**Comment by jat-247** [comment]
Hey @stilliard 
In fact this was not a false positive. I verified with Curl -I that the header was not being added due to a misconfiguration on the web hosts server (so my settings in the .htaccess file were correct but not being used by the server, yikes!). Thanks for such a quick response and an excellent tool for quick testing. I would not have known my host had screwed up security options without this. Closing this (non) issue.
